### PR TITLE
Adding a file for github recommendations.

### LINKED ISSUE #12: githubteacher recommendations for things to do - rlanda123
## dining
- Paul Martin's American Grill
- Flemming's Prime Steakhouse
- Rock and Brews
## Beaches nearby
1. El Segundo Beach
2. Manhattan Beach

### PR DESCRIPTION
**Pull request by rlanda123**
This will be a file on things to do in El Segundo and resolves #12

### CODE REVIEW

File: github-ravee-recommendations.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### El Segundo, CA
+
+### Places to Eat
```
Comment (yangj) [inline github-ravee-recommendations.md:3]: Any recommended places to eat?

Comment (rlanda123) [inline github-ravee-recommendations.md:3]: I like Chinees

File: github-ravee-recommendations.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Fun things to do
```
Comment (yangj) [inline github-ravee-recommendations.md:8]: @rlanda123 - you may want to fill this section up

File: github-ravee-recommendations.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+### Places to Stay
```
Comment (yangj) [inline github-ravee-recommendations.md:15]: @rlanda123 - same here

**Pull request merged**